pages/user/moderation.js

```diff
@@ -2207,45 +2207,58 @@ const UserModerationQueue = ({
+      value: 'requestEmailConfirmation',
+      label: 'Institution Email is added but not confirmed',
+      rejectionText: `Please confirm the institutional email in your profile by clicking "Confirm" button next to the email and enter the verification token received.\n\n${instructionText}`,
     },
     {
-      value: 'imPersonalHomepageAndEmail',
-      label: 'Impersonal Homepage + Missing Institution Email',
-      rejectionText: `A Homepage url which displays your name and institutional email matching your latest career/education history are required.\n\n${instructionText}`,
+      value: 'invalidDBLP',
+      label: 'DBLP is disambiguation page',
+      rejectionText: `The DBLP link you have provided is a disambiguation page and is not intended to be used as a bibliography. Please select the correct bibliography page listed under "Other persons with a similar name". If your page is not listed please contact the DBLP team so they can add your bibliography page. We recommend providing a different bibliography homepage when resubmitting to OpenReview moderation.\n\n${instructionText}`,
     },
     {
-      value: 'repeatedInvalidInfo',
-      label: 'Repeated Invalid Info',
-      rejectionText: `Submitting invalid info is a violation of OpenReview's Terms and Conditions which may result in terminating your access to the system.\n\n${instructionText}`,
+      value: 'imPersonalHomepage',
+      label: 'Homepage is invalid',
+      rejectionText: `The homepage url provided in your profile is invalid or does not display your name so your identity can't be determined.\n\n${instructionText}`,
     },
     {
-      value: 'requestEmailVerification',
-      label: 'Request Email Verification',
-      rejectionText: `Please add and confirm an institutional email to your profile.\n\n${instructionText}`,
+      value: 'imPersonalHomepageAndEmail',
+      label: 'Homepage is invalid + no institution email',
+      rejectionText: `A Homepage url which displays your name and institutional email matching your latest career/education history are required. Please confirm the institution email by entering the verification token received after clicking confirm button next to the institutional email.\n\n${instructionText}`,
     },
     {
-      value: 'invalidName',
-      label: 'Invalid Name',
-      rejectionText: `A valid name is required and must match the one listed on your provided personal homepages.\n\n${instructionText}`,
+      value: 'lastNotice',
+      label: 'Last notice before block',
+      rejectionText: `If invalid info is submitted again, your email will be blocked.\n\n${instructionText}`,
     },
   ]
 
+  const updateMessageForPastRejectProfile = () => {
+    setRejectionMessage(
+      (p) =>
+        `Submitting invalid info is a violation of OpenReview's Terms and Conditions which may result in terminating your access to the system.\n\n${p}`
```

Comment:
Can we hyperlink OpenReview's Terms and Conditions so users can check out the page?